cm_text/text_client.py

```diff
@@ -74,7 +115,7 @@ def send(self):
 
             # Send the message(s)
             try:
-                response = requests.post("https://gw.cmtelecom.com/v1.0/message", data=data, headers=headers)
+                response = requests.post(url=self.gateway, data=data, headers=headers)
             except Exception as e:
```

Comment:
Maybe an idea to rethrow this exception?

Reply:
We catch an error from the requests library that does the actual http call to our API. Expected errors could be the server is down or unreachable, or something along those lines.
By re-throwing our SDK users could catch the error and retry using a back-off mechanism or something, so it can't hurt to re-throw.